functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configurationfile/LocalChangesNewTestsSelectionExecutionWithConfigFileFunctionalTest.java

```diff
@@ -69,4 +70,25 @@ public void should_only_execute_new_tests_related_to_single_local_change_using_f
         assertThat(actualTestResults.accumulatedPerTestClass()).containsAll(expectedTestResults).hasSameSizeAs(expectedTestResults);
 
     }
+
+    @Test
+    public void should_load_configuration_from_parent_dir_if_not_present_in_current_execution_dir() throws Exception {
```

Comment:
Looking at the [tag commit](https://github.com/arquillian/smart-testing-dogfood-repo/commit/7b519516fd5e14ecb76a0cd67783293169715f26) - this is can replace the previous [test method](https://github.com/arquillian/smart-testing/pull/247/files#diff-34d1e596686e34c6dfc22210f43e6ffdR49) as it tests almost the same thing - there is no need to add a new test-method here and increase the test execution time
So just remove the test method `should_only_execute_new_tests_related_to_single_local_change_using_failsafe`